### PR TITLE
Add electrum segwit

### PR DESCRIPTION
**Pull request by junderw**
This branch is based on #1046 

So ideally we should merge #1046 first.

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/mx72h6fzwhxn5d0aw8dx1br8qc

[download apk](http://filestorage.bluewallet.io:1488/1588586547.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/c8th0yyygd0rmt28yen4dc8ak4

[download apk](http://filestorage.bluewallet.io:1488/1588631044.apk)

**Comment by junderw** [comment]
Rebased

**Comment by junderw** [comment]
2020-05-05 08-16-45 Screenshot

Nice

File: tests/unit/hd-segwit-electrum-seed-p2wpkh-wallet.test.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+describe('HDSegwitElectrumSeedP2WPKHWallet', () => {
+  it('can import mnemonics and generate addresses and WIFs', async function() {
+    let hd = new HDSegwitElectrumSeedP2WPKHWallet();
+    hd.setSecret('method goddess  humble  crumble output snake essay carpet monster barely trip betray ');
+    assert.ok(hd.validateMnemonic());
+
+    let address = hd._getExternalAddressByIndex(0);
+    assert.strictEqual(address, 'bc1q2yv6rhtw9ycqeq2rkch65sucf66ytwsd3csawr');
+
```
Comment (Overtorment) [inline tests/unit/hd-segwit-electrum-seed-p2wpkh-wallet.test.js:13]: assert for xpub would be nice

**Comment by junderw** [comment]
Fixed issues. Currently trying it out on Android Emulator

**Comment by junderw** [comment]
範囲を選択_003
範囲を選択_004

**Comment by junderw** [comment]
LGTM now.

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ubu3eqrgeqyzbyvkp89wyj5rmm

[download apk](http://filestorage.bluewallet.io:1488/1588683376.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/xunhdxyyw52eqzce5h3m9g12br

[download apk](http://filestorage.bluewallet.io:1488/1588683986.apk)

**Comment by Overtorment** [comment]
thanks!

**Pull request merged**